packages/provider/src/types.ts

```diff
@@ -217,10 +217,22 @@ export type DeleteThreadOptions = {
   force?: boolean,
 }
 
+export enum ThreadType {
```

Comment:
Enums are generally discouraged in TS (just one [example of why](https://www.aaron-powell.com/posts/2020-05-27-the-dangers-of-typescript-enums/))

So typically, a const object would be used instead which mostly has the same properties